app/controllers/login.rb

```diff
@@ -0,0 +1,29 @@
+get '/login' do
+  if session_logged_in?
```

Comment:
You should also make an `ensure_logged_in!` helper method that uses halt to redirect to `/login`